### PR TITLE
Additional management commands for bungiesearch

### PR DESCRIPTION
**Pull request by afrancis13**
Added some simple commands that are just aliases for combinations of previously existing management commands that would be commonly used in projects using Bungiesearch.

Added the logic to setup in the test to ensure that it's working properly. I've also tested this in an external application, with positive results.

### CODE REVIEW

File: bungiesearch/management/commands/clear_index.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+from django.core.management.base import BaseCommand
+
+from django.utils import six
+
+
+class Command(BaseCommand):
+    help = 'Clears the search index of its contents.'
+
+    option_list = BaseCommand.option_list + (
+        make_option('--noinput',
```
Comment (ChristopherRabotin) [inline bungiesearch/management/commands/clear_index.py]: Given how dangerous this is, what I tend to do have have two flags, usually the second one being `--guilty-as-charged`. It sounds a bit dumb and a bit of an overkill, but it saves data: https://github.com/Sparrho/bungiesearch/blob/master/bungiesearch/management/commands/search_index.py#L43 .

**Comment by ChristopherRabotin** [comment]
Looks good. Could you just add a `guilty-as-charged` flag to the clear index command, such as the flag defined here: https://github.com/Sparrho/bungiesearch/blob/master/bungiesearch/management/commands/search_index.py#L43 ? Thanks.

**Comment by afrancis13** [comment]
This is great. Was asleep when you initially commented, but looks like you got the flag merged in with a separate branch. Thanks!

**Comment by ChristopherRabotin** [comment]
You're welcome. I'm writing up the releases notes now so you can be the first to use version 1.2.0 ;)

**Comment by ChristopherRabotin** [comment]
@afrancis13 , version 1.2.0 is now on pypi.

Thanks again for all your contributions, I hope to see even more!

**Pull request merged**